DOC-3132: A document watermark can now be specified for the exported file (#3597)
tinymce/tinymce-docs: modules/ROOT/pages/7.7.0-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Export to Word

The {productname} {release-version} release includes an accompanying release of the **Export to Word** premium plugin.

**Export to Word**  includes the following fix.

==== A document watermark can now be specified for the exported file

> In previous versions of the Export to Word premium plugin, there was no watermark functionality, preventing users from adding watermarks to their exported Word documents.

Reviewer: ```suggestion
Previously, the **Export to Word** premium plugin did not support watermarks, preventing integrators from adding them to exported Word documents.
```

> {productname} {release-version} resolves this by introducing watermark functionality in the plugin, allowing users to add watermarks to their documents.

Reviewer: ```suggestion
With the release of {productname} {release-version} watermark functionality has been introduced, enabling watermarks to be included in exported documents.
```
